Qpf/Macro/Data/Constructors.lean

```diff
@@ -62,6 +71,6 @@ def mkConstructors (view : DataView) (shape : Name) : CommandElabM Unit := do
   let explicit ← view.getExplicitExpectedType
   let nameGen := (·.declName.replacePrefix (←getCurrNamespace) .anonymous)
 
-  mkConstructorsWithNameAndType view shape nameGen explicit
+  mkConstructorsWithNameAndType view shape nameGen explicit explicit #[]
```

Comment:
```suggestion
  mkConstructorsWithNameAndType view shape nameGen explicit explicit (binders := #[])
```

In functions with many arguments, I think it is sometimes confusing to track all the arguments by position. Thus, I think it's stylistically bettter to pass by named parameters for the arguements.